### PR TITLE
HL7 EDI Connector - Fixed the text about the required license

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

File: ldap/3.5/modules/ROOT/pages/ldap-connector-examples.adoc [hunk 1]
```diff
@@ -0,0 +1,277 @@
+= LDAP Connector Connector 3.5 Examples - Mule 4
```
Comment (kevintroller) [inline ldap/3.5/modules/ROOT/pages/ldap-connector-examples.adoc:1]: Remove duplicated "Connector" name in heading.

File: ldap/3.5/modules/ROOT/pages/ldap-connector-examples.adoc [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
++
+[source,text,linenums]
+----
+config.principal.dn=<DN>
+config.password=<Password>
+config.url=<URL>
+----
++
+. Place the `mule-artifact.properties` file into the project's `src/main/resources` directory.
+. In Studio, click *HTTP* and drag the *Listener* source to the canvas.
```
Comment (kevintroller) [inline ldap/3.5/modules/ROOT/pages/ldap-connector-examples.adoc]:
```suggestion
. From the *Mule Palette* view in Studio, click *HTTP* and drag the *Listener* source to the canvas.
```

File: ldap/3.5/modules/ROOT/pages/ldap-connector-examples.adoc [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+. On the Listener properties window, change the display name of the *Listener* source to `HTTP`.
+. Click *+* next to the *Connector configuration* field to add a global element.
+. Accept the defaults.
+. On the Listener properties window, set the *Path* field value to `/`.
```
Comment (kevintroller) [inline ldap/3.5/modules/ROOT/pages/ldap-connector-examples.adoc]:
```suggestion
. On the Listener properties window, set *Path* to `/`.
```

File: ldap/3.5/modules/ROOT/pages/ldap-connector-examples.adoc [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+	</flow>
+</mule>
+----
+
+[[set-up-common-truststore]]
+== Set Up a Common Truststore
+
+This example shows how to change the previous example so that it has a TLS configuration that uses a custom truststore.
+
+NOTE: The custom truststore feature is not FIPS compliant.
```
Comment (kevintroller) [inline ldap/3.5/modules/ROOT/pages/ldap-connector-examples.adoc:225]:
```suggestion
[NOTE]
The custom truststore feature is not FIPS compliant.
```

**Review by kevintroller — COMMENTED** [review]
Added minor suggestions

**Review by kevintroller — DISMISSED** [review]

**Review by kevintroller — APPROVED** [review]

**Review by tasmoore — APPROVED** [review]

**Pull request merged**